policyengine_canada/parameters/gov/cra/tax/income/credits/gst_credit/base_amounts.yaml

```
@@ -0,0 +1,19 @@
+description: This tax credit, introduced in 1986, is a refundable credit calculated by summing a base amount for the head, spouse, and number of children under 19, then reducing this amount by a proportion of 'family net income' as defined in the SPSD/M
+
+HEAD:
+  2022-07-01: 306
+SPOUSE:
+  2022-07-01: 306
+CHILD:
+  2022-07-01: 306
+
+metadata:
+  unit: currency-CAD
+  period: year
+  name: gst_credit_base_amount
+  label: GST Credit Base Amount
+  reference:
+    - title: Government of Canada - GST calculation sheet for the July 2022 to June 2023 payments (2021 base year)
+      href: https://www.canada.ca/content/dam/cra-arc/serv-info/benefits/gstcclc21-eng.pdf
+  breakdown:
+ 
```

Though could we check the legal code here? I wonder if CRA just breaks it down this way for clarity, while it might be a single number in the law (except for non-dependant children, whom we can ignore for now). If that's the case this can follow the law with one number.
```suggestion
   - gst_credit_category
```

---

Good call -- the SPSD/M Parameter Guide has all of the values going back to 1997 (citing either the law or the calculation sheet, depending on the year), and they only ever have one fixed number per year.

---

Just realized the SPSD/M seems to disregard non-child dependents for this. The documentation for the parameter they call `Federal sales tax credit amount for dependant` says it just applies to children. I also had input the wrong number for children (I had used the amount given for non-child dependents). Maybe we revise to align with the SPSD/M?

---

Yeah it looks like the SPSD/M differs from the gov't documentation: it doesn't distinguish between 'eligible dependents' and 'eligible non-dependent children'. 

The [gov't calculation sheet[(https://www.canada.ca/en/revenue-agency/services/child-family-benefits/goods-services-tax-harmonized-sales-tax-gst-hst-credit/goods-services-tax-harmonized-sales-tax-credit-calculation-sheet-july-2022-june-2023-payments-2021-tax-year.html) distinguishes between 4 classes of people:

- filer
- spouse
- 'eligible dependent'
- 'eligible children -- for children not included as eligible dependents'.

But the SPSD/M only has a separate 'amount' parameter for 3 types of people:

- FSTCF -- Federal sales tax credit amount for **filer**
- FSTCS -- Federal sales tax credit amount for **spouse**
- FSTCC -- Federal sales tax credit amount for **dependent**

The documentation for parameter FSTCC says: _"This amount is claimable for the Federal Sales Tax Credit on behalf of each child under the age of 18 years."_, but the 2022 parameter value ($161) corresponds to the amount on the calculation sheet for non-dependent children. 


I wonder why the SPSD/M chose to do this/maybe it's a mistake? Regardless I guess we'll need to choose whether to align with the SPSD/M or stay true to the calculation sheet.

---

Seems to be an SPSD/M error. Could we model all four categories for completeness? I think we might have non-dependant children elsewhere in the model for shared custody arrangements.

---

So I emailed the SPSD/M team about this and they replied pretty fast! They point out that the gov't calculation sheet is actually misleading: the category of 'eligible dependent' is only meant to allow a single-parent household to get the 'spouse' amount for one of the children. They reference an old edition of the calculation sheet that makes this more clear, and which I've attached to this comment (see specifically the table 3 on page 8)
[rc4210-21e.pdf](https://github.com/PolicyEngine/policyengine-canada/files/10386481/rc4210-21e.pdf)

---

Oh I see - Canada gives married benefits to single parents this way in a couple places, and I guess the lack of a plural in `Credit for eligible dependant` was a clue that was happening here too.